README.md

```diff
@@ -4,3 +4,35 @@ This repository contains ports of Tock OS (https://www.tockos.org) to Titan
 chips.
 
 This is not an officially supported Google product.
+
+
+## Getting started
+
+### Clone the repo
+
+Get the source:
+
+```shell
+git clone --recursive git@github.com:google/tock-on-titan.git
+```
+
+### Get the tools and libs to build the code
+
+Download Rust
+
+```shell
+cd tock-on-titan
+curl https://sh.rustup.rs -sSf | sh
```

Comment:
Could we also add that into the `setup` make target?

Reply:
I'd be mildly against doing that (but only mildly).

The `curl` command actually installs rust.  I think it's important for the user to be aware that they have done that.

The `rustup` commands that `make setup` runs install more rust packages, but it feels like a different thing to me, and running `make setup` does get across the idea that something like that is going on.

That being said, if you still think that the convenience of not typing a second command outweighs the loss of transparency, I'll make the change.  Let me know.

Reply:
Hmm.  I think I've convinced myself that you're right about this. I will make this change in the next round.

Reply:
See branch `dann/add-initial-rust-download-to-setup-target of https://github.com/dann21/tock-on-titan.git.  I will cut a PR for what's on this branch once this PR has been disposed of.

Reply:
Ah, good point re `rustup` installing a whole Rust environment. I guess being explicit (as you have it now) is better then. LG!

Reply:
I think we just swapped positions.  We can continue the discussion if/when I submit a PR for branch `dann/add-initial-rust-download-to-setup-target` of https://github.com/dann21/tock-on-titan.git. 

Heh.

Reply:
Is the `rustup` installation a no-op if you already have `rustup` installed? I believe `make setup` can safely be run multiple times (i.e. I think it should be idempotent).

Reply:
Yes.  Actually, to be fair, I'm not certain that it's a bit-for-bit no-op -- it may rerun the installation, copying new bits over the old bits.  So running it may refresh what's there.  But it won't delete anything, so it _appears_ idempotent to me.

Reply:
In any case, for this purpose, I'm pretty sure it's OK.

Reply:
We can carry this discussion further when I make a PR out of branch `dann/add-initial-rust-download-to-setup-target of https://github.com/dann21/tock-on-titan.git.  Coming soon.

Reply:
FYI, I did try this in a separate PR.  Mt attempt failed.  See #195 for details.